va/validation-authority.go

```diff
@@ -637,33 +641,75 @@ func (va *ValidationAuthorityImpl) checkCAARecords(ctx context.Context, identifi
 	if err != nil {
 		return
 	}
+
 	if caaSet == nil {
 		// No CAA records found, can issue
+		va.stats.Inc("VA.CAA.None", 1, 1.0)
 		present = false
 		valid = true
 		return
-	} else if caaSet.criticalUnknown() {
-		present = true
+	}
+
+	present = true
```

Comment:
This method is now way too long for named return values. Please swap those out. (It was already too long for them, anyway, and we're kind of mostly anti-them anyway)